sonic-chassisd/scripts/chassisd

```diff
@@ -138,6 +138,7 @@ class ModuleConfigUpdater(logger.Logger):
         super(ModuleConfigUpdater, self).__init__(log_identifier)
 
         self.chassis = chassis
+        self.lock = threading.Lock()
```

Comment:
Is this needed?

Reply:
Good catch! Looks like I forgot to clean it up.